packages/guideline-blocks-settings/src/components/BlockItemWrapper/Toolbar/Toolbar.spec.tsx

```diff
@@ -33,15 +34,89 @@ describe('Toolbar', () => {
     });
 
     it('should not throw error if toolbar does not have attachments enabled', () => {
-        expect(() =>
-            render(<Toolbar items={[]} flyoutMenu={{ items: [] }} attachments={{ isEnabled: false }} />),
-        ).not.toThrowError();
+        expect(() => render(<Toolbar items={[]} attachments={{ isEnabled: false }} />)).not.toThrowError();
     });
 
     it('should throw error if toolbar does have attachments enabled without provider', () => {
-        expect(() =>
-            render(<Toolbar items={[]} flyoutMenu={{ items: [] }} attachments={{ isEnabled: true }} />),
-        ).toThrowError();
+        expect(() => render(<Toolbar items={[]} attachments={{ isEnabled: true }} />)).toThrowError();
+    });
+
+    it('should render every item type', async () => {
```

Comment:
Can we have this also tested with Keyboard? as now its only tested on click?

Reply:
fixed

Reply:
Would split the ones for keyboard to an own test

Reply:
fixed